src/web/themes/custom/bcgov_career/templates/content/node--career-profile--full.html.twig

```diff
@@ -130,7 +131,7 @@
             </div>
             <div class="bottom-btn">
               <a class="btn btn-career-profile" href="{{ node.field_workbc_link.0.uri }}" target='_blank'>View Career Profile</a>
-              <a class="btn btn-find-job" href="{{ node.field_find_job.0.uri }}" target='_blank'>Find Jobs</a>
+              <a class="btn btn-find-job" href="{{ workbc_url }}/search-and-prepare-job/find-jobs#/job-search;noc={{ node.label }}" target='_blank'>Find Jobs</a>
```

Comment:
I am used to seeing a trailing semicolon `;` on that search URL. If you tested that it's not needed, then that's fine.

Reply:
fixed.

it does work without but may as well keep things consistent.